doc/source/examples/cd_clf_adult.ipynb

```diff
@@ -0,0 +1,434 @@
+{
```

Comment:
<p>I feel like a paragraph explaining the usefulness of calibration (either here or in the method description notebook, with a link to it from here) would be nice.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi-detect/pull/414/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>